Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -810,16 +795,6 @@ private bool NoPendingFileSystemRescan()
             return !this.fileSystemChanges.TryPeek(out var change) || change.Type != WatcherChangeTypes.All;
         }
 
-        internal void RegisterFileChangeNotification(FileNode node, FileSystemEventHandler handler)
-        {
-            this.fileChangedHandlers[node.Url] = handler;
-        }
-
-        internal void UnregisterFileChangeNotification(FileNode node)
-        {
-            this.fileChangedHandlers.Remove(node.Url);
-        }
```

Comment:
This was (unrelated) dead code?

Reply:
Related, because it's dead we no longer need to care about the filechanges.

Reply:
Unrelated to the first commit, related to the second.

---
In reply to: [159298890](https://github.com/Microsoft/nodejstools/pull/1827#discussion_r159298890) [](ancestors = 159298890)